schema.graphql

```diff
@@ -161,21 +176,6 @@ enum CalendarNameOptions {
   never
 }
 
-"""
-TODO
-"""
-type CheckboxWidget implements Component {
```

Comment:
This is a breaking change - like probably will crash the checklist app.

Are you gonna put console and checklist PRs up too?